WeakAuras/Init.lua

```diff
@@ -386,6 +384,34 @@ do
   end
   if WeakAuras.IsRetail() then
     tinsert(LibStubLibs, "LibSpecialization")
+    AddonCompartmentFrame:RegisterAddon({
+      text = AddonName,
+      icon = "Interface\\AddOns\\WeakAuras\\Media\\Textures\\icon.blp",
+      registerForRightClick = true,
+      registerForMiddleClick = true,
```

Comment:
These should just be `registerForAnyClick` by the way.

Reply:
Thanks!